addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -7884,6 +7939,16 @@ registry.BackgroundShape = SnippetOptionWidget.extend({
             return {showOnMobile: !this._getShapeData().showOnMobile};
         });
     },
+    /**
+     * Sets the animation ratio.
+     *
+     * @see this.selectClass for params
+     */
+    setAnimationRatio(previewMode, widgetValue, params) {
+        this._handlePreviewState(previewMode, () => {
+            return { animationRatio: widgetValue };
```

Comment:
Sadly it's now the Odoo js guideline (and the config of Odoo provided linter) :(
Same as using `"` everywhere instead of `'`

I guess those 2 ones have to be ignored as most of the employees and website colleagues are using it?